### PR TITLE
(GH-764) Redesign of Offline Page

### PR DESCRIPTION
**Pull request by designxsteph**
This is an enhancement and updates the look and feel of the offline page. This is one page that was not touched during the initial redesign. Some content has been slightly modified to be consistent with the rest of the website.

offline

### CODE REVIEW

File: chocolatey/Website/_app_offline.htm [hunk 1]
```diff
@@ -1,107 +1,80 @@
...
-        <div id="layout-footer">
-            <footer id="footer">
-                <p> &copy; 2011-2017 RealDimensions Software, LLC.</p>
-                <p> &copy; 2017-Present Chocolatey Software, Inc.</p>
-            </footer>
-        </div>
-    </body>
-</html>
+            <p>
+                We are working to make the Chocolatey Community Repository even more awesome and we’ll be back soon, but please don't hold your
```
Comment (ferventcoder) [inline chocolatey/Website/_app_offline.htm]: Chocolatey website and community repository

File: chocolatey/Website/_app_offline.htm [hunk 2]
```diff
@@ -1,107 +1,80 @@
...
+        a:hover{color:#2d79bb;}
+        blockquote{margin:0.923em;color:#666;font-style:italic;}
+        pre{margin:0.923em 0;}
+        pre,code,tt{font:1rem 'andale mono','lucida console',monospace;line-height:1.462em;}
+        .text-center{text-align:center;}
+    </style>
+</head>
+<body>
+    <header>
+        <h1>Chocolatey Community Repository</h1>
```
Comment (ferventcoder) [inline chocolatey/Website/_app_offline.htm]: Chocolatey Website and Community Repository

File: chocolatey/Website/_app_offline.htm [hunk 3]
```diff
@@ -1,107 +1,80 @@
...
+    </header>
+    <main>
+        <section>
+            <div class="text-center">
+                <h2>Chocolatey.org is currently offline</h2>
```
Comment (ferventcoder) [inline chocolatey/Website/_app_offline.htm]: The site is currently under maintenance/offline

File: chocolatey/Website/_app_offline.htm [hunk 4]
```diff
@@ -1,107 +1,80 @@
...
-</html>
+            <p>
+                We are working to make the Chocolatey Community Repository even more awesome and we’ll be back soon, but please don't hold your
+                breath we don't want you passing out at your desk and us getting sued, but I digress. If this outage is related to an incident
+                with the site, you can find more information on our <a href="https://status.chocolatey.org/">status page</a>, or you can
+                <a href="http://twitter.com/chocolateynuget">follow us on twitter</a> for updates or simply check back later. In the meantime
+                here are some things you might consider doing:
+            </p>
+            <ul>
+                <li>Beating that level you’re stuck on in Angry Birds</li>
```
Comment (ferventcoder) [inline chocolatey/Website/_app_offline.htm]: Remove

File: chocolatey/Website/_app_offline.htm [hunk 5]
```diff
@@ -1,107 +1,80 @@
...
+                <li>Solve world hunger, or at least get the kids more maps and stuff</li>
+                <li>Start crossing things off your Honey-Do List</li>
+                <li>Keep frantically hitting refresh waiting for Chocolatey.org to come back</li>
+            </ul>
+            <h3>Some more information:</h3>
+            <h4>What is the Chocolatey Community Repository?</h4>
+            <p>
+                You can develop your own package and share it via the Chocolatey Community Repository. Read the documentation for more details on
```
Comment (ferventcoder) [inline chocolatey/Website/_app_offline.htm]: This is a special section of the Chocolatey website that contains packages maintained and validated by our community that you can use for free to install and manage software. It's a great resource for the community, however if you are an organization there are some caveats to consider before using the community repository directly (such as rate limiting, reliability, etc).

You can also develop your own package and share it via the Chocolatey Community Repository.

**Comment by designxsteph** [comment]
@ferventcoder all changes have been made that were suggested!

**Comment by designxsteph** [comment]
@gep13 @vexx32 @ferventcoder I believe this PR is good to go. All suggestions have been implemented.

**Comment by gep13** [comment]
@designxsteph your changes have been merged, thanks for your contribution 👍

**Comment by gep13** [comment]
Fixes #764

**Pull request merged**